challenge.rb

```diff
@@ -1,26 +1,33 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+	input.map{|x| x.capitalize}
```

Comment:
can this be refactored? 
https://stackoverflow.com/questions/14881125/what-does-to-proc-method-mean